packages/core/src/common/disposable.ts

```diff
@@ -49,17 +49,20 @@ export class DisposableCollection implements Disposable {
         return this.disposables.length === 0;
     }
 
+    private onDisposeEnabled = true;
```

Comment:
`disposing`?

Reply:
maybe `disposingItems`? it's not reset as last statement of `dispose`.

Reply:
renamed to `disposingElements`